### ISSUE TITLE
bookmarklet does not hyphenate

### ISSUE THREAD

**Issue by allefeld** [robertknight/tex-linebreak#11]
I tried the bookmarklet on the linked Medium page, and while justification is applied and the line breaks look reasonably good, hyphenation is not enabled:

![Screenshot_20220213_003520](https://user-images.githubusercontent.com/1738353/153733383-2eb21813-3269-4d38-9674-f791a480c6d1.png)

Not sure if I misunderstood something, but the README says about the bookmarklet:

> It will justify and apply hyphenation to the content of any paragraph (`<p>`) elements on the page.

Google Chrome 98.0.4758.80 on Debian 11

**Comment by robertknight** [comment]
The library will only insert hyphens if it can't achieve acceptable results without them. If you make the viewport smaller, this is more likely to happen:

<img width="319" alt="Hyphenated article" src="https://user-images.githubusercontent.com/2458/153741411-95f10430-f76b-48fb-97d5-ba6179e7d78c.png">

**Comment by allefeld** [comment]
You're right. I guess I expected more hyphenations from my experience with LaTeX. Are the penalties the same as those used by TeX?

**Comment by robertknight** [comment]
> Are the penalties the same as those used by TeX?

The formula for scoring breakpoints is taken from the original [TeX algorithm](https://github.com/robertknight/tex-linebreak/blob/8085bec3e9f5d64dc190e703c3aaa86c799de146/src/layout.ts#L271), but I don't think the [cost values](https://github.com/robertknight/tex-linebreak/blob/8085bec3e9f5d64dc190e703c3aaa86c799de146/src/helpers.ts#L67) are the same. It is likely there are some other small differences too.
